project/.travis/before_install_test.sh.twig

```diff
@@ -6,9 +6,16 @@ if [ "${TRAVIS_PHP_VERSION}" != "hhvm" ]; then
     TRAVIS_INI_FILE="$PHP_INI_DIR/travis.ini"
     echo "memory_limit=3072M" >> "$TRAVIS_INI_FILE"
 
+    {% if 'mongodb' in services %}
     if [ "$TRAVIS_PHP_VERSION" '<' '7.0' ]; then
         echo "extension=mongo.so" >> "$TRAVIS_INI_FILE"
+    else
+        echo "extension=mongodb.so" >> "$TRAVIS_INI_FILE"
+
+        # Backwards compatibility with old mongo extension
+        composer require "alcaeus/mongo-php-adapter" --no-update
```

Comment:
Still thinking this can be added to `require-dev` of the project. Why not? This is also compatible with PHP 5.x.